pylintrc

```diff
@@ -235,7 +235,7 @@ notes=FIXME,XXX,TODO
 [SIMILARITIES]
 
 # Minimum lines number of a similarity.
-min-similarity-lines=5
+min-similarity-lines=10
```

Comment:
Is this change necessary any longer not that we've remove a lot of the SearchIndex stuff?